src/common.cpp

```diff
@@ -227,6 +236,14 @@ int shim::getrlimit(bionic::rlimit_resource res, bionic::rlimit *info) {
     return ret;
 }
 
+int shim::utimensat(int dirfd, const char *pathname, const struct timespec times[2], int flags) {
+#ifdef __APPLE__
+    return 0; // utimensat not available on all macOS versions
```

Comment:
`utimensat` is available on macOS
https://opensource.apple.com/source/xnu/xnu-4570.71.2/libsyscall/wrappers/utimensat.c
defined in sys/stat.h